src/renderers/common/Renderer.js

```diff
@@ -395,6 +395,12 @@ class Renderer {
 
 	}
 
+	async waitForGPUWorkDone() {
```

Comment:
Is it possible to shorten the name a bit? E.g. `waitForGPU()`?

Reply:
I don't mind but then I feel like we miss the concept of completion of task, such as `waitForGPUCompletion`, or `waitForGPUIdle`.

Reply:
Yes, `waitForGPUWorkDone()` is similar to the WebGPU api.

FWIW, other options are `waitForGPUResolve()`, `waitForGPUToResolve()`, or `waitForGPUToFullyResolve()`.

Reply:
My first thought was whether we could shorten the name too... I just think it's understanding that we're waiting for the GPU to finish tasks or be available?

```js
renderer.compute(veryHeavyComputeProgram)
await renderer.waitForGPU() // Ensures the CPU waits for the GPU to complete its operations
await renderer.backend.resolveTimestampAsync( veryHeavyComputeProgram, 'compute' );
```

Reply:
Let's go for `waitForGPU()` then!